### ISSUE TITLE
Getting blocked for 30min

### ISSUE THREAD

**Issue by anativ** [mgp25/Chat-API#1789]
Make sure these boxes are checked before submitting your issue -- thank you!

- [X] You are using PHP > 5.6
- [X] You have installed all extensions and modules listed in [here](https://github.com/mgp25/Chat-API/wiki/Dependencies). PHP Protobuf and Curve25519 are required!
- [X] You have checked if someone has already asked the same issue you have.
- [X] You have read the [wiki](https://github.com/mgp25/Chat-API/wiki).
- [X] Is not a programming question.
- [X] You are using latest Chat API code.

### Error
After a few minutes of running the code stops getting messages after a few minutes or restart 
A similar issue was posted in the past #1772 
I think it has something to do with the issue at Yowsup: https://github.com/jlguardi/yowsup/issues/44

**Comment by mgp25** [comment]
If you got blocked after 30 min, you are doing something wrong in your code, 100% sure

**Comment by anativ** [comment]
@mgp25 getting blocked not after 30min but for about 30min
Everything was ok for weeks till about 10 days ago... since then the code is unstable

**Comment by mgp25** [comment]
This code is stable what is not stable is the requests people makes in the scripts. If you follow exactly what is mentioned in readme, wiki and answered issues, you will see that everything works perfectly.

I have a number running with the API for 2 years now? Coincidence? I think not. It's very important to have a good code so the number can work without any problem.

Regards

**Comment by CodeTitanian** [comment]
Hi @mgp25 ,
I had a number that was working fine for about a year now. I used it for conversation in groups. It seems I have similar problem as @anativ , 
The system stops receiving messages every few minutes and after that, it starts getting messages again but only for limited amount of time. The issue started about two weeks ago. Since then, the code does not work as before. I'm familiar with the wiki and read it many times, I went over the issues that you closed and I don't know what has changed that creates this problem.

Thanks in advance for your response.
